astropy/io/votable/table.py

```diff
@@ -96,11 +97,11 @@ def parse(source, columns=None, invalid='exception', verify=None,
         VOTable specification (which is ``cds`` up to version 1.3 of
         VOTable, and ``vounit`` in more recent versions of the spec).
 
-    datatype_mapping : dict of str to str, optional
-        A mapping of datatype names to valid VOTable datatype names.
-        For example, if the file being read contains the datatype
-        "unsignedInt" (an invalid datatype in VOTable), include the
-        mapping ``{"unsignedInt": "long"}``.
+    datatype_mapping : dict, optional
+        A mapping of datatype names (str) to valid VOTable datatype names
```

Comment:
I'd write out `string` or back-quote the `str`.

Reply:
Doing the latter.

Reply:
✅